src/linagora.esn.calendar/app/app.js

```diff
@@ -102,7 +102,7 @@ require('esn-frontend-common-libs/src/frontend/js/modules/onscroll/on-scroll.mod
 require('esn-frontend-common-libs/src/frontend/js/modules/localstorage.js');
 require('esn-frontend-common-libs/src/frontend/js/modules/scroll.js');
 require('esn-frontend-common-libs/src/frontend/js/modules/previous-page.js');
-
+require('esn-frontend-mailto-handler/src/index.js');
```

Comment:
Then you must add `esn.mailto-handler` in the calendar app no?

Reply:
yes, it already exists but this require was missing during the explicit dependency phase.